src/views/characters/CharacterDetails.vue

```diff
@@ -1,5 +1,5 @@
 <script setup>

```

Comment:
Nice but I do miss a button to remove skill, items, conditions, events from a character.
But that might be better in a different issue